### PR TITLE
Drake 1 0 0

### PR DESCRIPTION
**Pull request by mavericklou**
In this pull request there is no more self download and self upgrade feature. Still using a downloaded jar to run drake

### CODE REVIEW

File: Library/Formula/drake.rb [hunk 1]
```diff
@@ -2,14 +2,24 @@
 
 class Drake < Formula
   desc "Data workflow tool meant to be 'make for data'"
-  homepage 'https://github.com/Factual/drake'
-  url 'https://github.com/Factual/drake/releases/download/v0.1.7/drake.jar'
-  version '0.1.7'
-  sha1 'c3092bd4e62effa38d4ade09bc191463b1c93c6a'
+  homepage "https://github.com/Factual/drake"
+  head "https://github.com/Factual/drake.git"
```
Comment (mikemcquaid) [inline Library/Formula/drake.rb]: put `head` below `sha1`

File: Library/Formula/drake.rb [hunk 2]
```diff
@@ -2,14 +2,24 @@
...
+  version "v1.0.0"
```
Comment (mikemcquaid) [inline Library/Formula/drake.rb]: put `version` below `sha1`

File: Library/Formula/drake.rb [hunk 3]
```diff
@@ -2,14 +2,24 @@
...
+  url "https://raw.githubusercontent.com/Factual/drake/master/bin/drake-pkg"
+  sha1 "3791f8e681620dae927bc70e22452b2703402724"
```
Comment (mikemcquaid) [inline Library/Formula/drake.rb]: use `sha256`, thanks!

**Comment by DomT4** [comment]
```
==> brew audit drake							 FAILED
Error: 3 problems in 1 formula
drake:
 * Stable: version v1.0.0 should not have a leading 'v'
 * Stable resource "jar": version v1.0.0 should not have a leading 'v'
 * JARs were installed to "/usr/local/Cellar/drake/v1.0.0/lib"
Installing JARs to "lib" can cause conflicts between packages.
For Java software, it is typically better for the formula to
install to "libexec" and then symlink or wrap binaries into "bin".
See "activemq", "jruby", etc. for examples.
The offending files are:
  /usr/local/Cellar/drake/v1.0.0/lib/drake-v1.0.0-standalone.jar
```

File: Library/Formula/drake.rb [hunk 4]
```diff
@@ -2,14 +2,24 @@
...
 class Drake < Formula
   desc "Data workflow tool meant to be 'make for data'"
-  homepage 'https://github.com/Factual/drake'
-  url 'https://github.com/Factual/drake/releases/download/v0.1.7/drake.jar'
-  version '0.1.7'
-  sha1 'c3092bd4e62effa38d4ade09bc191463b1c93c6a'
+  homepage "https://github.com/Factual/drake"
+  head "https://github.com/Factual/drake.git"
+  version "v1.0.0"
+  url "https://raw.githubusercontent.com/Factual/drake/master/bin/drake-pkg"
```
Comment (xu-cheng) [inline Library/Formula/drake.rb]: The URL is not versionized.

**Comment by mavericklou** [comment]
@DomT4 @xu-cheng @mikemcquaid 
Hey guys, can you check it again if this formula is good enough to be published?

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**